docs/Labeled.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: "Labeled"
+---
+
+# `<Labeled>`
```

Comment:
Could you update `Reference.md` reference of Labeled component to point to this new file?

Reply:
Done